shared/locales/bg/options.json

```diff
@@ -0,0 +1,91 @@
+{
+  "smartling" : {
```

Comment:
Can these live in a different repo? This will make history of the repo nothing related to code.

Reply:
Not sure I follow.. strings were part of the code before, and they still are.. just now localized.  Why is that unrelated to code?

Reply:
English ones yeah; as soon as you add every translation you're opening up to commit history polluted by way too many non-code commits.

Either we find a way to avoid that (via bundling them only at extension release time, or batching updates) or we create a submodule or other repo to hold these changes.